cli/args/mod.rs

```diff
@@ -66,12 +66,6 @@ use thiserror::Error;
 
 use crate::sys::CliSys;
 
-pub static DENO_DISABLE_PEDANTIC_NODE_WARNINGS: Lazy<bool> = Lazy::new(|| {
-  std::env::var("DENO_DISABLE_PEDANTIC_NODE_WARNINGS")
```

Comment:
Did we ever announce it publicly? Or has it just been moved?

Reply:
We never announced this one publicly.